omnigibson/utils/transform_utils_np.py

```diff
@@ -204,10 +212,8 @@ def quat_conjugate(quaternion):
     Returns:
         np.array: (x,y,z,w) quaternion conjugate
     """
-    return np.array(
-        (-quaternion[0], -quaternion[1], -quaternion[2], quaternion[3]),
-        dtype=np.float32,
-    )
+    n_dims = len(quaternion.shape)
+    return quaternion * np.array([-1.0, -1.0, -1.0, 1.0], dtype=np.float32).reshape([1] * (n_dims - 1) + [4])
```

Comment:
what is this reshape doing? are you trying to make this compatible with a batch of quaternions? IF so, can you also update the docstring?

Reply:
yes, great point -- updated!